README.md

```diff
@@ -191,6 +240,9 @@ Making all defaults explicit makes it possible to prevent unsafe changes when a
 
 **Some values must be carried forward from the previous configuration.**
 
+The DefaultNetwork type must not be changed. Once OpenShiftSDN or OVNKubernetes is set it is permanent.
+The serviceNetwork must not be changed. Any existing clusterNetworks definitions must not be changed.
```

Comment:
This doesn't really fit in this subsection, and it isn't needed anyway. The docs already say previously that "most of the operator's configuration parameters are not changeable".